openelevationservice/server/api/querybuilder.py

```diff
@@ -54,33 +141,53 @@ def line_elevation(geometry, format_out, dataset):
     Model = _getModel(dataset)
     
     if geometry.geom_type == 'LineString':
-        query_points2d = db.session\
-                            .query(func.ST_SetSRID(ST_DumpPoints(geometry.wkt).geom, 4326) \
-                            .label('geom')) \
-                            .subquery().alias('points2d')
+        num_points = db.session \
+                        .query(func.ST_NPoints(geometry.wkt)) \
+                        .scalar()
+        
+        if int(num_points) != 2:
+            raise InvalidUsage(400, 4002, "Actually, only LineString with exactly 2 points are supported!")
+        
+        lineLen = max(geometry.bounds[2] - geometry.bounds[0], geometry.bounds[3] - geometry.bounds[1])
```

Comment:
Could you assign the bounds to variables with descriptive names, so we know what [0], [1], [2] and [3] are, to make it more readable?

Reply:
Added comment line:
`# geometry.bounds = (minX, minY, maxX, maxY)`